posthog/errors.py

```diff
@@ -136,7 +136,7 @@ def look_up_error_code_meta(error: ServerException) -> ErrorCodeMeta:
     44: ErrorCodeMeta("ILLEGAL_COLUMN"),
     45: ErrorCodeMeta("ILLEGAL_NUMBER_OF_RESULT_COLUMNS"),
     46: ErrorCodeMeta("UNKNOWN_FUNCTION", user_safe=True),
-    47: ErrorCodeMeta("UNKNOWN_IDENTIFIER"),
+    47: ErrorCodeMeta("UNKNOWN_IDENTIFIER", user_safe=True),
```

Comment:
Well, we anyway sanitize them to avoid emitting the full query, but generally the ones about the query being wrong are safe and potentially useful, those about some cluster config are definitely off limits